build-info-extractor/src/testFixtures/java/org/jfrog/build/IntegrationTestsBase.java

```diff
@@ -306,4 +311,24 @@ public void setFiles(List<String> files) {
         }
     }
 
+    /***
+     * Returns a custom exception error to be thrown when repository creation/deletion fails.
+     * @param repo - repo name.
+     * @param creating - creation or deletion failed.
+     * @param statusCode - status code returned.
+     * @param reason - reason returned.
+     * @return - custom error message.
+     */
+    private String getRepositoryCustomErrorMessage(String repo, boolean creating, int statusCode, String reason) {
```

Comment:
I think using this syntax will be shorter:
```suggestion
    private String getRepositoryCustomErrorMessage(String repo, boolean creating, StatusLine statusLine) {
```